api/ProgressEvent.json

```diff
@@ -142,7 +142,8 @@
               "version_removed": "6"
             },
             "samsunginternet_android": {
-              "version_added": false
+              "version_added": true,
+              "version_removed": "1.0"
```

Comment:
This change is really weird.  Did this come from the mirroring script?  If so, can you file an issue?